spec/models/item_spec.rb

```diff
@@ -79,5 +92,12 @@
         expect(Item.five_popular('asc')).to_not include(item_3)
       end
     end
+    describe '#set_default_image' do
+      it 'sets a default image path before saving if one is not present' do
+        user = create(:merchant)
+        item = user.items.create(name: 'apple1', image_link: '', inventory: 3, description: 'apple one', current_price: 200, enabled: true)
+        expect(item.image_link).to eq('https://picsum.photos/200/300?image=0')
```

Comment:
Might want to add a test here to make sure it doesn’t add the default image to an item that does already have an image link

Reply:
I add a different image in the first test - should I add an `expect(page).to_not have` for the default image on that test?

Reply:
I added a separate test for creating a good item without an image, and also added an expectation to the first test to expect to not see the default image. =]